fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -4102,6 +4102,8 @@ struct StartFullRestoreTaskFunc : RestoreTaskFuncBase {
 		}
 
 		state bool logsOnly = wait(restore.onlyAppyMutationLogs().getD(tr, false, false));
```

Comment:
Above this, after line `4095`, there should be a `tr->reset();` after committing. (This is a bug that was around before this PR).

Reply:
Good catch, thanks, done.